src/SourceDialog/SourceDialog.vala

```diff
@@ -24,7 +24,7 @@ public class Maya.View.SourceDialog : Gtk.Grid {
 
     private Gtk.Entry name_entry;
     private bool set_as_default = false;
-    private string hex_color = "#da3d41";
+    private string hex_color = "#ed5353";
```

Comment:
I'm not sure about this change.
If the change is good, would a shift to @STRAWBERRY_500 be more appealing here?